lib/mutant/expression.rb

```diff
@@ -2,83 +2,21 @@ module Mutant
 
   # Abstract base class for match expression
   class Expression
-    include AbstractType, Adamantium::Flat, Concord::Public.new(:match)
+    include AbstractType, Adamantium::Flat
 
-    include Equalizer.new(:syntax)
+    fragment             = /[A-Za-z][A-Za-z\d_]*/.freeze
```

Comment:
I know we talked about this as a way to define truly "private" constants, but it still seems weird to me.